pkg/translator/zipkin/zipkinv2/to_translator_test.go

```diff
@@ -45,6 +45,37 @@ func TestZipkinSpansToInternalTraces(t *testing.T) {
 			td:   generateTraceSingleSpanErrorStatus(),
 			err:  nil,
 		},
+		{
+			name: "span status",
+			zs: []*zipkinmodel.SpanModel{
+				{
+					SpanContext: zipkinmodel.SpanContext{
+						TraceID: convertTraceID(
+							[16]byte{0xF1, 0xF2, 0xF3, 0xF4, 0xF5, 0xF6, 0xF7, 0xF8, 0xF9, 0xFA, 0xFB, 0xFC, 0xFD, 0xFE, 0xFF, 0x80}),
+						ID: convertSpanID([8]byte{0xAF, 0xAE, 0xAD, 0xAC, 0xAB, 0xAA, 0xA9, 0xA8}),
+					},
+					Name: "min.data",
+					Tags: map[string]string{
+						"otel.status_code": "Ok",
```

Comment:
are you going to test Unset and Error as well?

Reply:
Sounds reasonable.

Reply:
Yay!